user-access-handlers/src/main/java/no/unit/nva/handlers/IdentityServiceInitHandler.java

```diff
@@ -98,26 +101,39 @@ private List<RoleDto> createDefaultRoles() {
         var creator = RoleDto.newBuilder().withRoleName(ROLE_ACQUIRED_BY_ALL_PEOPLE_WITH_ACTIVE_EMPLOYMENT)
                           .build();
         var curator = RoleDto.newBuilder().withRoleName("Curator")
-            .withAccessRights(
-                List.of(APPROVE_DOI_REQUEST,
-                    REJECT_DOI_REQUEST,
-                    READ_DOI_REQUEST,
-                    EDIT_OWN_INSTITUTION_RESOURCES,
-                    APPROVE_PUBLISH_REQUEST))
-            .build();
+                          .withAccessRights(
+                              List.of(APPROVE_DOI_REQUEST,
+                                      REJECT_DOI_REQUEST,
+                                      READ_DOI_REQUEST,
+                                      EDIT_OWN_INSTITUTION_RESOURCES,
+                                      APPROVE_PUBLISH_REQUEST))
+                          .build();
         var institutionAdmin = RoleDto.newBuilder().withRoleName("Institution-admin")
-            .withAccessRights(List.of(EDIT_OWN_INSTITUTION_RESOURCES,
-                EDIT_OWN_INSTITUTION_PROJECTS,
-                EDIT_OWN_INSTITUTION_USERS))
-            .build();
+                                   .withAccessRights(List.of(EDIT_OWN_INSTITUTION_RESOURCES,
+                                                             EDIT_OWN_INSTITUTION_PROJECTS,
+                                                             EDIT_OWN_INSTITUTION_USERS))
+                                   .build();
+        var importCandidateCurator =
+            RoleDto.newBuilder()
+                .withRoleName("Import-candidate-curator")
```

Comment:
Should use same naming convention on all Roles
either `Curator-import-candidate` or change `Curator-thesis` and `Curator-thesis-embargo`

Reply:
yes, noticed just after I request your review myself.